face_client/js/actions/task.js

```
@@ -100,21 +143,32 @@ async function deleteTask(index) {
     const task = {
         taskStatus: document.getElementById('taskStatusInput').value,
         taskDesc: document.getElementById('taskDescInput').value,
         taskName: document.getElementById('taskNameInput').value,
+        estimateHours: parseInt(document.getElementById('estimateHoursInput').value, 10),
+        logHours: parseInt(document.getElementById('logHoursInput').value, 10),  
     };
 
+    if (isNaN(task.estimateHours)) {
+        task.estimateHours = 0;
+    }
+    if (isNaN(task.logHours)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)